draft-ietf-tls-tls13.md

```diff
@@ -3385,12 +3395,14 @@ L zeroes.
        allow_dhe_resumption(2),
        allow_psk_resumption(4)
      } TicketFlags;
-     
+
+     opaque Ticket<1..2^16-1>;
+  
      struct {
          uint32 ticket_lifetime;
          uint32 flags;
          TicketExtension extensions<2..2^16-2>;
-         opaque ticket<0..2^16-1>;
+         Ticket tickets<2..2^16-1>;
```

Comment:
The smallest permitted contents of tickets would be 3 octets, since a Ticket must have at least one octet of content.